notebooks/WET/WET-003/WET-003-ComCam.ipynb

```diff
@@ -7,217 +7,757 @@
    "source": [
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #78.    plot = create_interactive_scatter()
</pre><p>Looking at the resulting plot there's  pileup at various field distances . What's the cause of that ? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-sitcom/ts_aos_analysis/pull/33/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>

Reply:
For instance 
<img width="391" alt="image" src="https://github.com/user-attachments/assets/2f331070-51ad-4fec-be6e-aa2d04d4973a" />
, but also 
![Uploading image.png…]()